src/Renderer/ThreeExtended/FirstPersonControls.js

```diff
@@ -122,6 +42,7 @@ class FirstPersonControls extends THREE.EventDispatcher {
      * @param {number} options.verticalFOV - define the max visible vertical angle of the scene in degrees (default 180)
      * @param {number} options.panoramaRatio - alternative way to specify the max vertical angle when using a panorama.
      * You can specify the panorama width/height ratio and the verticalFOV will be computed automatically
+     * @param {boolean} options.disableEventListeners - if true, the controls will not self listen to mouse/key events
```

Comment:
maybe we can complete this documentation, at least listing `onMouseDown`, etc... 

It's obvious when you see the source code, but I expect that when one does set this option, they will still need these `onMouse` fns somewhere. Listing them here will save them a trip to the source code and make this option even more explanatory.